test/tests/dash_c.py

```diff
@@ -0,0 +1,18 @@
+import sys
+import subprocess
+
+me = sys.executable
+
+with open('/dev/null')as ignore:
+    # We don't (yet?) require exact stderr or return code compatibility w/
+    # python. So we just check that we succeed or fail as appropriate.
+    def do(args):
+        print 0 == subprocess.call([me] + args, stderr=ignore)
+
+    run(["-c", "print 2 + 2"])
```

Comment:
Oops! Yeah, having an annotation for "this file is expected to return nonzero" might be nice. I'll try implementing that.